### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by calumabarnett**
This PR makes a number of changes to the user guidance.

It is primarily focussed on updating and restructuring the git and GitHub section following changes to how this is managed by the Operations Engineering Team.

However, there are also changes to other sections. These changes are primarily focussed on structure, rather than content, and on removing sections that are no longer relevant.

---

### git and GitHub

The git and GitHub section has been entirely restructured. Some content has been rewritten and some new content has been added:

- content has been moved from a single page to multiple pages
- there is a new summary page for the section as a whole – this gives a brief overview of git and GitHub and signposts users to get started
- the 'set up GitHub' page reuses mostly existing content with a few minor updates
- the 'create a new project in GitHub' page has largely been rewritten to make the process clearer
- the 'manage access in GitHub' page is new and contains information relating to updated processes
- the 'collborate on a project' page, the 'install packages from GitHub' page and pages on working with git in RStudio, JupyterLab and the command line reuse existing content with some minor updates
- the 'security on GitHub' page contains updated content and has a new section on what to do when you've accidentally pushed data to GitHub
- the 'GitHub organisation management' page is new and contains information about how the organisation is now managed, including processes and automation
- the 'Learning resources' page has been refreshed and updated

### Apps

The separate pages on RShiny apps and static webapps have been combined into a single 'apps' section.

The notice regarding new app deployments has been updated.

Guidance on deploying new apps has been removed as it is no longer relevant. Information has been retained where it relates to managing existing apps.

### About

Two separate pages containing information about the Analytical Platform have been combined into one. All the content remains the same.

### Other

Links to the #analytical-platform-support channel have been updated following its renaming.

### CODE REVIEW

File: source/documentation/get-started.md [hunk 1]
```diff
@@ -65,7 +65,11 @@ Some 2FA applications and QR scanners are unable to scan the QR code generated o
 
 You can re-enable your “Dark” Mode, Extension or Settings once this process has been completed.
 
-## 2. Analytical Platform Account Access
+### Read more about working with GitHub in MoJ
+
+We use GitHub to store code and collborate on it with others. You should read more about how to work with [git and GitHub](github) on the Analytical Platform. You will need to complete some additional steps before you can start working with it with RStudio or JupyterLab.
```
Comment (rossjones) [inline source/documentation/get-started.md:70]:
```suggestion
We use GitHub to store code and collborate on it with others. You should read more about how to work with [git and GitHub](https://github.com) on the Analytical Platform. You will need to complete some additional steps before you can start working with it with RStudio or JupyterLab.
```

File: source/documentation/github/create-project.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+You can also use:
+
+- [pull requests](https://docs.github.com/en/pull-requests) to propose changes to your code and receive reviews from others
+- [issues](https://docs.github.com/en/issues) to plan and track your work
+
+### Create a new repository
+
+To create a new repository, you should:
+
+1. Nativate to the [MoJ Analytical Services](https://github.com/moj-analytical-services) GitHub organisation page.
```
Comment (rossjones) [inline source/documentation/github/create-project.md:14]:
```suggestion
1. Navigate to the [MoJ Analytical Services](https://github.com/moj-analytical-services) GitHub organisation page.
```

File: source/documentation/get-started.md (hunk 1 above)
Comment (rossjones) [reply source/documentation/get-started.md:70]: After reading further, found that suggestion is invalid.

**Review by rossjones — APPROVED** [review]
Looks good, one small typo that leapt out and so highlighted as a suggested fix.

File: source/documentation/github/create-project.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+- [issues](https://docs.github.com/en/issues) to plan and track your work
+
+### Create a new repository
+
+To create a new repository, you should:
+
+1. Navigate to the [MoJ Analytical Services](https://github.com/moj-analytical-services) GitHub organisation page.
+2. In the repositories section, select **New**.
+3. Configure the properties of your repository, in line with the information below.
+4. Select **Create repository**.
```
Comment (NickWalt01) [inline source/documentation/github/create-project.md:17]: Is it worth adding a note to use a template repository at this point, I've copied in our template to AP Org, it gives some useful notes in the README to help setup repo https://github.com/moj-analytical-services/template-repository

File: source/documentation/github/create-project.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+If you accidentally create a repository in your personal GitHub account or a different organisation, you can [transfer it](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#transferring-a-repository-owned-by-your-personal-account) to the MoJ Analytical Services GitHub organisation.
+
+#### Repository name
+
+When choosing a name for your repository, we recommend:
+
+- keeping it short and simple
+- making it descriptive and meaningful
+- avoiding abbreviations and acronyms
+- using dashes to separate words
```
Comment (NickWalt01) [inline source/documentation/github/create-project.md:32]: An extra line to say "Use dashes not full stops in repo name" ?

**Review by NickWalt01 — APPROVED** [review]

**Comment by NickWalt01** [comment]
@calumabarnett the tech-doc-publisher version is old in the repo and URL check is does it hiding errors. In the latest version there are 400 URL errors which are a mix internal and external URL's

**Comment by NickWalt01** [comment]
Some URL's in the attached file are to private and internal GH repo's these will return a failed URL: [doc_errors.txt](https://github.com/moj-analytical-services/user-guidance/files/10556100/doc_errors.txt)

File: source/documentation/github/create-project.md (hunk 3 above)
Comment (calumabarnett) [reply source/documentation/github/create-project.md:17]: Added in 473de87

File: source/documentation/github/create-project.md (hunk 4 above)
Comment (calumabarnett) [reply source/documentation/github/create-project.md]: Updated in 6cd7aa6

**Comment by calumabarnett** [comment]
> Some URL's in the attached file are to private and internal GH repo's these will return a failed URL: [doc_errors.txt](https://github.com/moj-analytical-services/user-guidance/files/10556100/doc_errors.txt)

@NickWalt01 I'll pick this up in a separate PR so I can get this in.

**Pull request merged**